blueprints/addon/files/.travis.yml

```diff
@@ -27,8 +27,8 @@ before_install:
   - npm config set spin false
   - npm install -g bower
   - bower --version
-  - npm install phantomjs-prebuilt
-  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
+  - npm install -g phantomjs-prebuilt
```

Comment:
can you combine that with the `npm install -g bower` call above? no need to invoke NPM twice :)